frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.tsx

```diff
@@ -32,7 +33,11 @@ const MultiColumnField: React.FC<MultiColumnFieldProps> = ({
           helperText={helpText}
           isRequired={required}
         >
-          <MultiColumnFieldHeader headers={headers} />
+          {fieldValue.length < 1 ? (
+            <div className="text-secondary">{emptyMessage}</div> || null
```

Comment:
Try using the pf4's `Text` component here

Reply:
Not sure `Text` will work because there's only one variant that has the same secondary color which is `small`. I don't think we want smaller text here though.

Reply:
Now, using the `SecondaryStatus` component